src/test/java/api/support/http/InterfaceUrls.java

```diff
@@ -124,6 +124,10 @@ public static URL itemsInTransitReportUrl() {
     return circulationModuleUrl("/inventory-reports/items-in-transit");
   }
 
+   static URL pickSlipsReportUrl(String servicePointId) {
+    return circulationModuleUrl("/inventory-reports/pick-slips"  + servicePointId);
```

Comment:
Personally (assuming pick slips are considered an inventory report), I’d extract a method for an inventory reports path and then use a format for the rest of the path. Similar to the `requestQueueUrl` method below.